src/components/MultiHopTrade/hooks/useTradeExecution/useTradeExecution.ts

```diff
@@ -155,6 +179,7 @@ export const useTradeExecution = ({
           stepIndex: activeStepOrDefault,
           slippageTolerancePercentageDecimal,
           evm,
+          cow,
           utxo,
           cosmosSdk,
```

Comment:
Non-blocking, we might want to update this interface to nest the props, maybe something like the below instead of passing them all in at the top level:

```ts
tradeExecutionProps: {
    evm: EvmTradeExecutionProps
    cow: CowTradeExecutionProps
    utxo: UtxoTradeExecutionProps
    cosmosSdk: CosmosSdkTradeExecutionProps 
  }
```